### ISSUE TITLE
[bug] Bulk upload - Accepted formats link not working

### ISSUE THREAD

**Issue by kennymc-c** [unfoldedcircle/bug-reports#98]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The accepted formats link in the bulk upload window is not working respectively shows an error that this site doesn't exist:

![Bildschirmfoto 2023-10-01 um 20 38 15](https://github.com/unfoldedcircle/feature-and-bug-tracker/assets/14995555/b7b0676b-4394-41e5-a922-e8b5bf5f2e24)

https://support.unfoldedcircle.com/support/solutions/articles/101000455137-supported-infrared-formats

### How to Reproduce

1. In the web configurator go to Remotes/+/Create blank
2. Ente a name and click on Create
3. On the new page click on the pencil at the top left next to the name you just entered
4. Click on the Bulk import button on the lower right corner of the window
5. Click on the 'Click here' link (https://support.unfoldedcircle.com/support/solutions/articles/101000455137-supported-infrared-formats)

### Expected behavior

The link should show a website that lists all accepted ir formats

### System version

1.3.3

### What part of the system affected by the problem?

Web Configurator

**Comment by kennymc-c** [comment]
Fixed as the new web configurator doesn't include this link anymore

**Comment by WillTriumph1** [comment]
So there is no bulk upload supported?

**Comment by kennymc-c** [comment]
No, this issue was just about the non working link in the bulk upload menu. Bulk upload is still possible.
